File: Loops/FizzBuzz.c
#include <stdio.h> 
#include <string.h>
int i; 
int usr;
char one;
char two;
int main() {
    printf("Give me a number");
    scanf("%d", &usr); 
    printf("Give me a short word!\n");
    scanf("%s", &one); 
    printf("Give me another short word!\n");
    scanf("%s", &two); 
    //loop that counts to 50
  for(i=0;i<=usr;i++){
        
        //replace #'s divisable by 3 and 5 with "fizzbuzz"
    if(i%3==0 && i%5==0){
        strcat(one,two);
        printf("%s\n", one);

    }else if (i%3==0){ 
        
        //replace #'s divisable by 3 with "Fizz"
       printf("%s\n", two);
    }else if(i%5==0){
        
        //replace #'s divisable by 5 with "buzz"
        printf("Buzz\n");
    }else { 
        
        //print the # 
    printf("%d\n", i); 
    } 
   } return 0;
} 

File: Loops/FizzBuzz.py
user = int(input("give me a number\n"))
user1 = input("give me a short word\n")
user2 = input("give me another short word\n") 

three = user1 + user2 

for i in range(1, user + 1):
    if i % 3 == 0 and i % 5 == 0:
      print(three)
    elif i % 3 ==0:
        print(user1)
    elif i % 5 ==0:
        print(user2)
    else: 
        print(i) 